ui/app/common/concept-set/services/conceptGroupFormatService.js

```diff
@@ -40,11 +40,13 @@ angular.module('bahmni.common.conceptSet')
                 });
                 var obsValueList = [];
                 sortedGroupMembers.forEach(function (obs) {
-                    if (obs.value && obs.value.name) {
-                        obsValueList.push(obs.value.name);
-                    }
-                    else {
-                        obsValueList.push(obs.value);
+                    if (obs.concept.conceptClass !== "Abnormal") {
+                        if (obs.value && obs.value.name && obs.concept.conceptClass != "Abnormal") {
```

Comment:
Is obs.concept.conceptClass != "Abnormal" condition needed inside if as we are already have the same outside if?

Reply:
@SanoferSameera This helps the function ignore concepts of class "Abnormal" from getting appended to the value that gets displayed.